music21/romanText/translate.py

```diff
@@ -438,6 +438,8 @@ def translateOneLineToken(self, lineToken: rtObjects.RTTagged):
                 environLocal.warn(f'Could not parse RTVersion tag: {lineToken.data!r}')
 
         elif isinstance(lineToken, rtObjects.RTTagged):
+            # We could add this to md as custom metadata:
+            # md.addCustom('romantext:' + lineToken.tag, lineToken.data)
             otherMetadata = RomanTextUnprocessedMetadata(lineToken.tag, lineToken.data)
             self.p.append(otherMetadata)
```

Comment:
better not to, because these tend to represent the state at the particular part of the piece not at the beginning.  In fact, I think that `movementNumber` also is something where I've seen multiple movements inscribed in one piece, and should probably be a separate MD at the current parse position if parse position != offset(0).  But that's for another time.  Good to leave this alone for now.  Later we can do the code to put additional Metadata elements at different parts of the piece.

Reply:
Removed this comment.